### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by davidworn**
Rails setup + other logistics stuff + testing out pull request workflow.

### CODE REVIEW

**Comment by davidworn** [comment]
This is beautiful code. You are a genius.

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ source 'https://rubygems.org'
    gem 'sqlite3'
  end
 
+ gem 'bootstrap-sass'
```
Comment (davidworn) [inline Gemfile]: I had an issue with this and needed to move it to *before* the bootstrap gem. Odd.

File: Gemfile [hunk 2]
```diff
@@ -21,6 +22,7 @@ source 'https://rubygems.org'
  gem 'figaro'
  gem 'pry'
 
+
```
Comment (davidworn) [inline Gemfile]: Our coding style is for one line only of white space.

**Comment by dothework** [comment]
@davidworn Good here. PRess ahead.

**Pull request merged**